django/db/models/base.py

```diff
@@ -1142,23 +1142,22 @@ def _save_table(
                         ),
                     )["_order__max"]
                 )
-            fields = [
+            insert_fields = [
```

Comment:
I changed the variable name as it made it much easier for me to reason about why the origin of the `f.generated and (pk_set or f is not meta.auto_field)` check.